packages/core/package.json

```diff
@@ -61,5 +61,8 @@
   "volta": {
     "extends": "../../package.json"
   },
-  "sideEffects": false
+  "sideEffects": false,
+  "devDependencies": {
+    "zod": "^3.24.1"
```

Comment:
Wasn't sure if adding deps was okay, but it made adding tests a lot easier

Reply:
This is fine for testing